sites/all/themes/orwell/templates/menu_block/menu-block-wrapper.tpl.php

```diff
@@ -0,0 +1,24 @@
+<?php
+/**
+ * @file
+ * Default theme implementation to wrap menu blocks.
```

Comment:
Think that this file comment should be updated, as it's not the default.

It should be `@{inherited}` and say what the main diffrent is form the on that comes with drupal core.